updater/updater.go

```diff
@@ -133,6 +135,9 @@ func Run(config *config.UpdaterConfig, datastore database.Datastore, st *utils.S
 					nextUpdate = lockExpiration
 				}
 			}
+		} else {
+			DbFinishUpdate = false
```

Comment:
@Quentin-M The `firstUpdate` variable can not be used to determine when the update is done, that is the reason I used another variable that will change the value depending on the stage of the update.